server/id/id_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+package id
+
+import (
+	"context"
+	"path"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/CeresDB/ceresmeta/server/etcdutil"
+	"github.com/CeresDB/ceresmeta/server/storage"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	clientv3 "go.etcd.io/etcd/client/v3"
+	"go.etcd.io/etcd/server/v3/embed"
+)
+
+const defaultRequestTimeout = time.Second * 10
+
+func TestAlloc(t *testing.T) {
```

Comment:
Need a concurrent test case or at least a TODO for that.

Reply:
TODO